source/auth/auth.rst

```diff
@@ -1250,45 +1250,44 @@ mechanism
     MUST be "MONGODB-OIDC"
 
 mechanism_properties
-    PROVIDER_NAME
-        The name of a built-in OIDC provider integration to use to obtain
-        credentials. The value MUST be one of ["aws"]. If both ``PROVIDER_NAME``
-        and an `OIDC Callback`_ are provided for the same ``MongoClient``
-        (either via the ``CALLBACK`` mechanism property or a ``MongoClient``
-        configuration), the driver MUST raise an error.
-
-    CALLBACK
-        An `OIDC Callback`_ that returns credentials for OIDC providers that do
-        not have a built-in integraiton. Drivers MAY allow the user to specify
-        an `OIDC Callback`_ using a ``MongoClient`` configuration instead of a
-        mechanism property, depending on what is conventional for the driver.
-        Drivers MUST NOT support both the ``CALLBACK`` mechanism property and
-        the ``MongoClient`` configuration.
-
-    CALLBACK_TYPE
-        The type of `OIDC Callback`_. The value MUST be one of ["machine",
-        "human"]. Drivers MUST NOT allow the user to specify ``CALLBACK_TYPE``
-        in the connection string. Drivers MAY allow the user to specify the
-        callback type using a ``MongoClient`` configuration instead of a
-        mechanism property to be consistent with how the `OIDC Callback`_ is
-        configured. If an `OIDC Callback`_ is configured and ``CALLBACK_TYPE``
-        is not specified, the driver MUST raise an error. This property is only
-        required for drivers that support the `Human Authentication Flow`_.
-
-    ALLOWED_HOSTS
-        The list of allowed hostnames or ip-addresses (ignoring ports) for
-        MongoDB connections. The hostnames may include a leading "\*." wildcard,
-        which allows for matching (potentially  nested) subdomains.
-        ``ALLOWED_HOSTS`` is a security feature and MUST default to
-        ``["*.mongodb.net", "*.mongodb-qa.net", "*.mongodb-dev.net",
-        "*.mongodbgov.net", "localhost", "127.0.0.1", "::1"]``. When
-        MONGODB-OIDC authentication using a `Human Callback`_ is attempted
-        against a hostname that does not match any of list of allowed hosts, the
-        driver MUST raise a client-side error without invoking any user-provided
-        callbacks. This value MUST NOT be allowed in the URI connection string.
-        The hostname check MUST be performed after SRV record resolution, if
-        applicable. This property is only required for drivers that support the
-        `Human Authentication Flow`_.
+  PROVIDER_NAME
+    Drivers MUST allow the user to specify the name of a built-in OIDC provider
+    integration to use to obtain credentials. If provided, the value MUST be one
+    of ["aws"]. If both ``PROVIDER_NAME`` and an `OIDC Callback`_ or `Human
```

Comment:
Should we call `Human Callback` `OIDC Human Callback` to be consistent with the mechanism property name?